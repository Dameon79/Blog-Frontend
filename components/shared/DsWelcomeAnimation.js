import { Component } from 'react'

class DsWelcomeAnimation extends Component {
  render () {
    return  <div className='a-ds-welcome-animation'>
      <div className='a-ds-welcome-animation__cell1'>D</div>
      <div className='a-ds-welcome-animation__cell1'>A</div>
      <div className='a-ds-welcome-animation__cell1'>M</div>
      <div className='a-ds-welcome-animation__cell1'>E</div>
      <div className='a-ds-welcome-animation__cell1'>O</div>
      <div className='a-ds-welcome-animation__cell1'>N</div>
      <div className='a-ds-welcome-animation__cell1'></div>
      <div className='a-ds-welcome-animation__cell1'></div>
      <div className='a-ds-welcome-animation__cell1'></div>
      <div className='a-ds-welcome-animation__cell2'>S</div>
      <div className='a-ds-welcome-animation__cell2'>U</div>
      <div className='a-ds-welcome-animation__cell2'>M</div>
      <div className='a-ds-welcome-animation__cell2'>P</div>
      <div className='a-ds-welcome-animation__cell2'>T</div>
      <div className='a-ds-welcome-animation__cell2'>O</div>
      <div className='a-ds-welcome-animation__cell2'>N</div>
      <div className='a-ds-welcome-animation__cell2'></div>
      <div className='a-ds-welcome-animation__cell2'></div>
      <div className='a-ds-welcome-animation__cell3'>D</div>
      <div className='a-ds-welcome-animation__cell3'>E</div>
      <div className='a-ds-welcome-animation__cell3'>V</div>
      <div className='a-ds-welcome-animation__cell3'>E</div>
      <div className='a-ds-welcome-animation__cell3'>L</div>
      <div className='a-ds-welcome-animation__cell3'>O</div>
      <div className='a-ds-welcome-animation__cell3'>P</div>
      <div className='a-ds-welcome-animation__cell3'>E</div>
      <div className='a-ds-welcome-animation__cell3'>R</div>
    </div>
  }
}

export default DsWelcomeAnimation