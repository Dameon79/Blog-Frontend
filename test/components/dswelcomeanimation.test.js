import DsWelcomeAnimation from '../../components/shared/DsWelcomeAnimation'

describe('<DsWelcomeAnimation />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <DsWelcomeAnimation />
    )

    expect(component).toMatchSnapshot()
  })
})