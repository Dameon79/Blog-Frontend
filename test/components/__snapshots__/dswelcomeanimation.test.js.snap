// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DsWelcomeAnimation /> it renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DsWelcomeAnimation />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="a-ds-welcome-animation__cell1"
        >
          D
        </div>,
        <div
          className="a-ds-welcome-animation__cell1"
        >
          A
        </div>,
        <div
          className="a-ds-welcome-animation__cell1"
        >
          M
        </div>,
        <div
          className="a-ds-welcome-animation__cell1"
        >
          E
        </div>,
        <div
          className="a-ds-welcome-animation__cell1"
        >
          O
        </div>,
        <div
          className="a-ds-welcome-animation__cell1"
        >
          N
        </div>,
        <div
          className="a-ds-welcome-animation__cell1"
        />,
        <div
          className="a-ds-welcome-animation__cell1"
        />,
        <div
          className="a-ds-welcome-animation__cell1"
        />,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          S
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          U
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          M
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          P
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          T
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          O
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        >
          N
        </div>,
        <div
          className="a-ds-welcome-animation__cell2"
        />,
        <div
          className="a-ds-welcome-animation__cell2"
        />,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          D
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          E
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          V
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          E
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          L
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          O
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          P
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          E
        </div>,
        <div
          className="a-ds-welcome-animation__cell3"
        >
          R
        </div>,
      ],
      "className": "a-ds-welcome-animation",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "D",
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": "D",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "A",
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": "A",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "M",
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": "M",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "E",
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": "E",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "O",
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": "O",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "N",
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": "N",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "a-ds-welcome-animation__cell1",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "S",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "S",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "U",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "U",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "M",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "M",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "P",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "P",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "T",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "T",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "O",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "O",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "N",
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": "N",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "a-ds-welcome-animation__cell2",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "D",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "D",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "E",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "E",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "V",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "V",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "E",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "E",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "L",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "L",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "O",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "O",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "P",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "P",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "E",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "E",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "R",
          "className": "a-ds-welcome-animation__cell3",
        },
        "ref": null,
        "rendered": "R",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="a-ds-welcome-animation__cell1"
          >
            D
          </div>,
          <div
            className="a-ds-welcome-animation__cell1"
          >
            A
          </div>,
          <div
            className="a-ds-welcome-animation__cell1"
          >
            M
          </div>,
          <div
            className="a-ds-welcome-animation__cell1"
          >
            E
          </div>,
          <div
            className="a-ds-welcome-animation__cell1"
          >
            O
          </div>,
          <div
            className="a-ds-welcome-animation__cell1"
          >
            N
          </div>,
          <div
            className="a-ds-welcome-animation__cell1"
          />,
          <div
            className="a-ds-welcome-animation__cell1"
          />,
          <div
            className="a-ds-welcome-animation__cell1"
          />,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            S
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            U
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            M
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            P
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            T
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            O
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          >
            N
          </div>,
          <div
            className="a-ds-welcome-animation__cell2"
          />,
          <div
            className="a-ds-welcome-animation__cell2"
          />,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            D
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            E
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            V
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            E
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            L
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            O
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            P
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            E
          </div>,
          <div
            className="a-ds-welcome-animation__cell3"
          >
            R
          </div>,
        ],
        "className": "a-ds-welcome-animation",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "D",
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": "D",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "A",
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": "A",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "M",
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": "M",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "E",
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": "E",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "O",
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": "O",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "N",
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": "N",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "a-ds-welcome-animation__cell1",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "S",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "S",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "U",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "U",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "M",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "M",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "P",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "P",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "T",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "T",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "O",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "O",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "N",
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": "N",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "a-ds-welcome-animation__cell2",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "D",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "D",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "E",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "E",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "V",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "V",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "E",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "E",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "L",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "L",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "O",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "O",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "P",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "P",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "E",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "E",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "R",
            "className": "a-ds-welcome-animation__cell3",
          },
          "ref": null,
          "rendered": "R",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
